### PR TITLE
Update types.ts to fix wrong void return type of multiGet

### PR DESCRIPTION
**Pull request by mtt87**
Fix https://github.com/react-native-async-storage/async-storage/issues/764

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -125,7 +125,7 @@ export type AsyncStorageStatic = {
   multiGet: (
     keys: string[],
     callback?: MultiGetCallback
-  ) => Promise<readonly KeyValuePair[] | void>;
+  ) => Promise<readonly KeyValuePair[]>;
```
Comment (krizzu) [inline src/types.ts:128]: you'll also need to update the type of [returned promise here](https://github.com/react-native-async-storage/async-storage/blob/master/src/AsyncStorage.native.ts#L272)

Comment (mtt87) [reply src/types.ts:128]: Thank you I missed it 😄

**Comment by krizzu** [comment]
There's one last bit that needs to be one - [updating the type of MultiRequest type](https://github.com/react-native-async-storage/async-storage/blob/master/src/types.ts#L26).

```diff
-  resolve?: (result?: readonly KeyValuePair[]) => void;
+  resolve?: (result: readonly KeyValuePair[]) => void;
```

**Review by tido64 — APPROVED** [review]

**Comment by tido64** [comment]
Sorry I couldn't find time for this. I'm glad that you two managed to figure this out 😄

**Review by krizzu — APPROVED** [review]
Great 👏

**Pull request merged**